README.md

```diff
@@ -12,24 +12,32 @@ Access planetary datasets from the Planetary Data System (PDS)
 
 See https://nasa-pds.github.io/peppi/
 
-### Use as MCP server with Claude (alpha)
 
-A specific command line can be used to connect peppi (and the PDS API to an LLM), this has been tested with Claude.
+### Use as MCP server with Claude (alpha)
 
-Use command: `pds-peppi-mcp-server`
+Two commands enable the connection of AI apps with Peppi using the [Model Context Protocol](https://modelcontextprotocol.io/) (MCP) using the MCP `stdio` transport:
 
-Connect it to Claude Desktop, for example, as described in https://modelcontextprotocol.io/quickstart/user#installing-the-filesystem-server, using the following configuration:
+- `pds-peppi-mcp-server` is a proof-of-concept MCP server that enables natural-language based tools for searches for instrument hosts and targets
```

Comment:
Before we merge, I would prefer that we only keep one MCP server, otherwise that would be unclear for a user where to start. Would we be able to merge the capabilities of both in one ? Or would the one you created already cover all the feature of the initial MCP server ? I might discover that soon as I am starting the review the PR.

Reply:
@tloubrieu-jpl oh okay, sure one is probably better than two especially if one can do the job of both

Are there some queries that one can answer better than the other? I'm not what you would call a "domain expert" 😉